web/docs/advanced/links.md

```diff
@@ -69,18 +107,36 @@ const linkToTask = routes.TaskRoute.build({ params: { id: 1 } })
 
 This will result in a link like this: `/task/1`.
 
-You can also pass `search` and `hash` props to the `build` function. Check out the [API Reference](#routes-object) for more details.
+### Optional Static Segments
```

Comment:
Why is this repeated twice?

Reply:
It seems they both ended up in the guide:
![image](https://github.com/user-attachments/assets/70534cb2-8966-4817-ac97-d859582192e2)

I don't a chapter dedicated to it in the API reference. Maybe you mispositioned it?

Reply:
Oh my bad, I had a brain fart it seems. The reason this is repeated is to explain the same concept in two different contexts: `Link` component and the `routes` object.

It's a bit repetitive, I agree, but it's easier to follow if the structure is the same in both places so users can better pattern match "ah yes, that's the same thing". At least that's my goal here.